composer.json

```diff
@@ -56,7 +56,7 @@
     },
     "minimum-stability" : "dev",
     "require": {
-        "oat-sa/lib-tao-qti" : "~2.0.0",
+        "oat-sa/lib-tao-qti" : "dev-feature/TAO-3410-flysystem-test-state-files",
```

Comment:
After experimentation, will have to refer to "oat-sa/lib-tao-qti":"~2.1.0" before merging.